packages/bot/src/database/get_user.ts

```
@@ -27,13 +27,32 @@ import {
+	getCharaInMemory,
+	getTemplateByInteraction,
+	mergeAttribute,
+	updateMemory,
+} from "database";
 import type { EmbedBuilder, Message } from "discord.js";
 import * as Djs from "discord.js";
 import equal from "fast-deep-equal";
-import { embedError, ensureEmbed, getEmbeds, reply } from "messages";
+import { embedError, ensureEmbed, getEmbeds, reply, replyEphemeralError } from "messages";
 import { isSerializedNameEquals, searchUserChannel } from "utils";
+import {StatisticalTemplate} from "@dicelette/core";
```

Biome (2.3.13)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)